### PR TITLE
fix the analyticstack box

### PR DESCRIPTION
**Pull request by mulby**
Merge the changes in #3369 into master as well.

Reviewers: @edx/devops @nedbat

### CODE REVIEW

File: vagrant/base/analyticstack/Vagrantfile [hunk 1]
```diff
@@ -90,5 +95,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     if ENV['ECOMMERCE_VERSION']
       ansible.extra_vars['ECOMMERCE_VERSION'] = ENV['ECOMMERCE_VERSION']
     end
+    if ENV['ECOMMERCE_WORKER_VERSION']
+      ansible.extra_vars['ECOMMERCE_WORKER_VERSION'] = ENV['ECOMMERCE_WORKER_VERSION']
+    end
+    if ENV['PROGRAMS_VERSION']
+      ansible.extra_vars['PROGRAMS_VERSION'] = ENV['PROGRAMS_VERSION']
+    end
```
Comment (brianhw) [inline vagrant/base/analyticstack/Vagrantfile:103]: nit:  Just curious why only less than half of the versions have an override mechanism, and the others do not.

File: vagrant/release/analyticstack/Vagrantfile [hunk 2]
```diff
@@ -65,14 +70,11 @@ end
...
+  "open-release/eucalyptus" => {
+    :name => "eucalyptus-analyticstack-2016-09-04", :file => "eucalyptus-analyticstack-2016-09-04.box",
   },
-  "named-release/dogwood.2" => {
-    :name => "analyticstack", :file => "dogwood-analyticstack-2016-03-15.box",
+  "open-release/eucalyptus.2" => {
+    :name => "eucalyptus-analyticstack-2016-09-04", :file => "eucalyptus-analyticstack-2016-09-04.box",
   },
   "named-release/dogwood.3" => {
     :name => "analyticstack", :file => "dogwood-analyticstack-2016-03-15.box",
```
Comment (brianhw) [inline vagrant/release/analyticstack/Vagrantfile:80]: nit: out of curiosity, do we ever just call it "dogwood", the way we do with eucalyptus?

**Review by brianhw — APPROVED** [review]
LGTM.  👍

File: vagrant/release/analyticstack/Vagrantfile (hunk 2 above)
Comment (nedbat) [reply vagrant/release/analyticstack/Vagrantfile:80]: Dogwood had releases "named-release/dogwood", then "named-release/dogwood.1", .2, etc. This was confusing.

Eucalyptus started with "open-release/eucalyptus.1", then "open-release/eucalyptus.2", etc.

Or maybe this isn't what you meant?

**Comment by nedbat** [comment]
Can we get this merged? I have confirmed that these are the same changes as in #3369. :+1:

**Comment by nedbat** [comment]
@mulby ping ping

**Comment by mulby** [comment]
I'm still waiting for @edx/devops to give a thumb

**Comment by nedbat** [comment]
BTW, the two pull-request numbers are super-confusing, but very cool. Nice work!

**Comment by nedbat** [comment]
I agree, it would be cool to get a thumb from @edx/devops ....

File: vagrant/base/analyticstack/Vagrantfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Vagrant.require_version ">= 1.6.5"
+Vagrant.require_version ">= 1.5.3"
```
Comment (jibsheet) [inline vagrant/base/analyticstack/Vagrantfile]: Intentional Vagrant version downgrade?

**Comment by thallada** [comment]
@mulby Anything holding this back? Without this update, the analyticstack install will fail when I follow the readthedocs instructions.

**Comment by thallada** [comment]
I was wrong. Even with this Vagrantfile, I get a migration error in Insights while running `vagrant up`.

~~Also, isn't this diff basically the same as the one in this merged PR? https://github.com/edx/configuration/pull/3369~~

~~What happened? Did it get reverted?~~ Nevermind: I see that PR was merging into a different branch.

**Comment by mulby** [comment]
@thallada and I looked into this earlier today, I suspect the issue is that I haven't built a base vagrant box for master yet

File: vagrant/base/analyticstack/Vagrantfile (hunk 3 above)
Comment (mulby) [reply vagrant/base/analyticstack/Vagrantfile]: hrmmm... nope

**Comment by nedbat** [comment]
Any plans to land this?

File: vagrant/base/analyticstack/Vagrantfile (hunk 3 above)
Comment (mulby) [reply vagrant/base/analyticstack/Vagrantfile]: Fixed!

Comment (mulby) [reply vagrant/base/analyticstack/Vagrantfile]: @jibsheet can you review again?

**Review by jibsheet — APPROVED** [review]

**Pull request merged**